### DISCUSSION TITLE
Manually sort recipes

### DISCUSSION THREAD

**Discussion by ken8521** [mealie-recipes/mealie#3099] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I tried searching and didn't see this mentioned, so hopefully it's not a dupe.  It's not really a problem, just exactly what it is, a feature request.

As it is recipes are listed in the order they were added when you are looking at "All Recipes".  I'd like a way to manually order them.

### Please provide a concise description of the feature that would resolve your issue.

Sort of giving my recipes funny names (001, 002, etc..) and sorting them by name.. I'm not seeing anyway to just manually sort.

### Please consider and list out some caveats or tradeoffs made in your design decision

Can't really think of any.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by RealFoxie** [comment]
Any reason why you need this specifically?
could be implemented by allowing filling in a hidden sorting field, or by having a Ui that let's you sort everything, but I wonder how this can ever be manageable if you have more than a few of them.

**Comment by ken8521** [comment]
Just preference I guess.  It's mundane I admit.  It just bugs me not being able to order them as I prefer... Edit:  If I could sort by category, that would be fine to.  I just don't like having a Breakfast Burrito recipe next to a an Apple Cobbler recipe, because that is the order they were added in.
